### PR TITLE
Added CONTRIBUTING.md

### LINKED ISSUE #79: please create contributing.md
### Description

It would be nice if there is the instruction given to run the project.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by alokVerma749**
`Closes #79 ` 

## Fixes Issue

Closes #79 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [**not**] My change requires changes to the documentation.
- [**not**] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/128"></a>

### CODE REVIEW

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Contributors
+
+## 💥 How to Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/EddieHubCommunity/LinkFree/pulls)
```
Comment (emmalearnscode) [inline CONTRIBUTING.md:5]: Linkfree needs changing to good first issues in the link

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+```bash
+git remote add upstream https://github.com/EddieHubCommunity/good-first-issue-finder.git
+```
+
+**5.** Create a new branch:
+
+```bash
+git checkout -b YourBranchName
+```
+
```
Comment (emmalearnscode) [inline CONTRIBUTING.md:42]: Add a step here to explain how to add the secret github token in a .env file to get the project up and running

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+**7.** Make your changes to the source code.
+
+**8.** Stage your changes and commit:
+
+⚠️ **Make sure** not to commit `package.json` or `package-lock.json` file
+
+⚠️ **Make sure** not to run the commands `git add .` or `git add *`. Instead, stage your changes for each file/folder
+
+```bash
+git add public
```
Comment (emmalearnscode) [inline CONTRIBUTING.md:69]: This is specific to adding profiles in Linkfree. It might be better to write git add filename

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Looks like a great start. Just a few things that need updating as they refer to the Linkfree repo and aren't relevant here.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

File: CONTRIBUTING.md (hunk 1 above)
Comment (alokVerma749) [reply CONTRIBUTING.md]: I fixed this.

File: CONTRIBUTING.md (hunk 2 above)
Comment (alokVerma749) [reply CONTRIBUTING.md:42]: Added the required step.

File: CONTRIBUTING.md (hunk 3 above)
Comment (alokVerma749) [reply CONTRIBUTING.md]: I fixed this.

**Review by emmalearnscode — APPROVED** [review]
Thanks, LGTM!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**